manifests/i/Intel/PresentMon/Beta/0.5.0.0/Intel.PresentMon.Beta.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Intel.PresentMon.Beta
+PackageVersion: 0.5.0.0
+PackageLocale: en-US
+Publisher: Intel
+PackageName: PresentMon Beta

```

```suggestion
PackageName: Intel PresentMon
```
This should match control panel

---

I just used the other Intel's software example, but sure.